assets/opstools/AppBuilder/classes/RowFilter.js

```diff
@@ -1628,9 +1628,13 @@ export default class RowFilter extends OP.Component {
 
 			},
 
-			inQueryValid: function(rowData, rule, compareValue) {
+			inQueryValid: function(rowData, columnName, rule, compareValue) {
 
 				let result = false;
+				
+				if (columnName) {
+					rowData = rowData[columnName] || [];
```

Comment:
Should the default value be Object ? 

Normally, rowData is a Object except when it is a value of M:N relation

Reply:
Good catch.